api/SourceBuffer.json

```diff
@@ -477,10 +518,10 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": "3.0"
+              "version_added": false
             },
             "webview_android": {
-              "version_added": "4.4.3"
+              "version_added": false
```

Comment:
I argued against it, but Mozilla's policy is to leave pre-Chromium WebView data as is.

Reply:
This was wrong, though. There cannot have been a WebView version which had this enable by default, if I'm reading the version history of WebKit and Chromium correctly.

Is the policy around WebView stated somewhere? Is it just that the pre-Chromium releases are preserved in browsers/webview_android.json aren't being removed, or actually that we should keep whatever the data is, even when incorrect?

Reply:
This is incorrect:

> Mozilla's policy is to leave pre-Chromium WebView data as is

#4479 covers the introduction of `"≤37"` values for WebView. The summary is that we decided to allow ≤37 while continuing to permit more precise versioning (37 or earlier) if a contributor could substantiate it. We didn't freeze any data.

If the feature's unsupported now and you can't find any evidence that it was supported historically and the data conflicts with it, then the data should change.

Reply:
Thanks @ddbeck. I'm quite confident this data was wrong, because WebView 4.4.3 was using Chromium M33, and the code for this feature didn't even exist until M51.